Build doesn't work locally with recent node/npm versions

The build doesn't appear to work with recent node/npm versions. With node v7.10.0 and npm v4.5.0, I get the dreaded `Error: Cannot find module 'internal/fs'`. I think I've traced this down to gitbook 2.6.7 which seems to rely on older versions of node and graceful-fs. On the other hand upgrading gitbook to 3 breaks restructure-navigation which seems to be fixed to gitbook 2.x (upgrading all other dependencies to their latest seems to be ok, including grunt). I also  tried the restructure-navigation3 version in npm and had no luck. It looks like [restructure-navigation](https://github.com/zalando-incubator/gitbook-restructure-navigation) is something Zalando built for itself (and perhaps for these guidelines, not sure). I haven't tried a workaround with moving the global npm/node versions back yet, but that's suboptimal for contributors.

This isn't breaking builds for publishing, but gitbook 3's been out for a while and staying on old versions in the node/npm ecosystem is usually unwise.

--- a/BUILD.md
+++ b/BUILD.md
@@ -1,5 +1,9 @@
 # Build
 
+## Requirements
+
+You have to use node version 5.1.1 until we upgrade to current Gitbook version.
+
 ## Install all dependencies
 
     npm install

--- a/README.md
+++ b/README.md
@@ -15,6 +15,7 @@ E-Book Support
 -----
 
 You can easily generate PDF, ePub, Mobi files out of our guidelines. Please refer to this [explanation](https://toolchain.gitbook.com/ebook.html) - you have to install Callibre.
+You have to use node version 5.1.1 to build the guidelines.
 CAUTION: you need to add version 2.6.7 identifier to every build command:
 
   ```bash

